pkg/tests/api/alerting/api_alertmanager_configuration_test.go

```diff
@@ -71,12 +76,15 @@ func TestAlertmanagerConfigurationIsTransactional(t *testing.T) {
 func TestAlertmanagerConfigurationPersistSecrets(t *testing.T) {
 	dir, path := testinfra.CreateGrafDir(t, testinfra.GrafanaOpts{
 		EnableFeatureToggles: []string{"ngalert"},
-		AnonymousUserRole:    models.ROLE_EDITOR,
+		DisableAnonymous:     true,
 	})
 
 	store := testinfra.SetUpDatabase(t, dir)
+	// override bus to get the GetSignedInUserQuery handler
```

Comment:
I'd love to see a test where we have two different configurations for two users from different organisations.

Reply:
I have added the test (together with some fixes/improvements)!